src/components/command-bar/commands/chat/index.tsx

```diff
@@ -31,9 +31,9 @@ const Icon = () => {
 }
 const chatCommand: CommandBarCommand = {
 	name: 'chat',
-	icon: <Icon />,
+	icon: <BackButton />,
 	inputProps: {
-		placeholder: () => 'Return to search',
+		placeholder: () => 'Hide me',
```

Comment:
This seems to have to be there but also doesn't seem to do anything because it's not actually in input. If there is a way to do this better that would be good.